### PR TITLE
Hopefully clarify some things in the README

### PR DESCRIPTION
**Pull request by workflow**
Let me know if I'm stating something incorrectly!

cc: @dingxiangfei2009 @trha @antigravityla

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,19 @@ bringing you reproducible builds and better caching.
...
+- A utility written in Rust to generate version pins of crate dependencies
   and boilerplates
+  
+Together, these components will take an existing Cargo.lock and delegate the process of fetching and combining your dependencies (generated by cargo) using the deterministic nix package manager.
 
 
 ## Build instructions
 
-1. Install the [Nix package manager](https://nixos.org/nix) on your *NIX
+1. Install the [nix package manager](https://nixos.org/nix) on your *nix
```
Comment (ebkalderon) [inline README.md]: The package manager name "nix" should be capitalized.

File: README.md [hunk 2]
```diff
@@ -5,17 +5,19 @@ bringing you reproducible builds and better caching.
 
 This repository hosts two components:
 
-- a [Nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at `overlay`
+- A [nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at the `/overlay`
```
Comment (ebkalderon) [inline README.md]: The name "nixpkgs" should be capitalized for consistency with the official Nix documentation.

File: README.md [hunk 3]
```diff
@@ -5,17 +5,19 @@ bringing you reproducible builds and better caching.
...
   directory, providing utilities to build your Cargo workspace and set up
   airplane mode.
   
-- a utility written in Rust to generate version pins of dependency crates
+- A utility written in Rust to generate version pins of crate dependencies
   and boilerplates
+  
+Together, these components will take an existing Cargo.lock and delegate the process of fetching and combining your dependencies (generated by cargo) using the deterministic nix package manager.
```
Comment (ebkalderon) [inline README.md]:
```suggestion
Together, these components will take an existing `Cargo.lock` and delegate the process of fetching and combining your dependencies (generated by Cargo) using the deterministic Nix package manager.
```

File: README.md (hunk 1 above)
Comment (workflow) [reply README.md]: Done :)

File: README.md (hunk 2 above)
Comment (workflow) [reply README.md]: Done :)

**Pull request merged**